src/server/api/campaign-contact.js

```diff
@@ -65,24 +65,40 @@ export const resolvers = {
     campaign: async (campaignContact, _, { loaders }) => (
       loaders.campaign.load(campaignContact.campaign_id)
     ),
+    // To get that result to look like what the original code returned 
+    // without using the outgoing answer_options array field, try this:
+    //
     questionResponses: async (campaignContact, _, { loaders }) => {
 
-      const results = await r.table('question_response')
-        .getAll(campaignContact.id, { index: 'campaign_contact_id'})
-        .eqJoin('interaction_step_id', r.table('interaction_step'))
-        .zip()
-        .innerJoin(r.table('interaction_step'), function(left, right) {
-          return left('interaction_step_id').eq(right( 'parent_interaction_id'))
-        })
+      const results = await r.knex('question_response as qres')
+        .where('question_response.campaign_contact', campaignContact.id)
+        .join('interaction_step', 'qres.interaction_step_id', 'interaction_step.id')
+        .join('interaction_step as child',
```

Comment:
Joins answer options. So the results should include question_response : question_step : answer_option_1, question_response : question_step : answer_option_2, etc for each answer option.